pom.xml

```diff
@@ -16,7 +16,7 @@
     </scm>
 
     <properties>
-        <target.platform>oxygen</target.platform>
+        <target.platform>photon</target.platform>
```

Comment:
I would not change that, indeed it makes sense to set this to the minimum (!) target supported. This should be the same value used in the .setup file as the default value of the `eclipse.target.platform` variable.

I did not find any reference to what minimum Eclipse version we support. I would suggest using some mildly recent on like Mars or Neon.

Reply:
@erwint ok I will keep that to oxygen then :)